pkgs/applications/misc/googleearth/default.nix

```diff
@@ -20,60 +23,81 @@ let
     libXi
     libXv
     mesa
-    libXrender
-    libXrandr
-    libXfixes
+    libX11
     libXcursor
-    libXinerama
-    freetype
     libXext
-    libX11
-    qt4
+    libXfixes
+    libXinerama
+    libXrandr
+    libXrender
+    qtbase qtdeclarative qtlocation qtmultimedia qtscript qtsensors qtsvg qtwebchannel qtwebkit qtx11extras
     zlib
-    fontconfig
+    alsaLib
+    fontconfig freetype
   ];
-in
-stdenv.mkDerivation rec {
-  version = "7.1.4.1529";
+
+in stdenv.mkDerivation rec {
   name = "googleearth-${version}";
+  version = "7.3.0.3832";
 
   src = fetchurl {
-    url = "https://dl.google.com/earth/client/current/google-earth-stable_current_${arch}.deb";
+    url = "https://dl.google.com/earth/client/current/google-earth-pro-stable_current_${arch}.deb";
```

Comment:
Is it possible to use a mirror here to prevent the silent updates by Google in future?